hivemq-edge/src/frontend/src/extensions/datahub/designer/data_policy/DataPolicyPanel.tsx

```diff
@@ -0,0 +1,45 @@
+import { FC, useMemo } from 'react'
+import { Node } from 'reactflow'
+import { Card, CardBody } from '@chakra-ui/react'
+
+import { DataPolicyData, PanelProps } from '@datahub/types.ts'
+import useDataHubDraftStore from '@datahub/hooks/useDataHubDraftStore.ts'
+import { ReactFlowSchemaForm } from '@datahub/components/forms/ReactFlowSchemaForm.tsx'
+import { MOCK_DATA_POLICY_SCHEMA } from '@datahub/designer/data_policy/DataPolicySchema.ts'
+import { CustomValidator } from '@rjsf/utils'
+import { useGetAllDataPolicies } from '@datahub/api/hooks/DataHubDataPoliciesService/useGetAllDataPolicies.tsx'
+import { useTranslation } from 'react-i18next'
+
+export const DataPolicyPanel: FC<PanelProps> = ({ selectedNode, onFormSubmit }) => {
+  const { t } = useTranslation('datahub')
+  const { nodes } = useDataHubDraftStore()
+  const { data: allPolicies } = useGetAllDataPolicies()
+
+  const data = useMemo(() => {
+    const adapterNode = nodes.find((e) => e.id === selectedNode) as Node<DataPolicyData> | undefined
```

Comment:
suggestion(non-blocking): my classic comment :)

```suggestion
    const adapterNode = nodes.find((node) => node.id === selectedNode) as Node<DataPolicyData> | undefined
```